cmd/controller-manager/main.go

```diff
@@ -79,6 +79,11 @@ func main() {
 		klog.Fatal("NAMESPACE environment variable not set")
 	}
 
+	helmRelease := os.Getenv("HELM_RELEASE")
+	if helmRelease == "" {
+		klog.Fatal("HELM_RELEASE environment variable not set")
```

Comment:
Is it OK that if this env is not set, we just use the original endpoint name in L194?